docs/reference/intro.md

```diff
@@ -3,8 +3,34 @@ sidebar_position: 1
 id: intro
 ---
 
-# Intro
+# Welcome to OpenFeature!
 
-Welcome to OpenFeature docs!
+OpenFeature is an open standard for feature flagging, created to support a robust feature flag ecosystem using cloud native technologies.
 
-OpenFeature is an open standard for feature flag management, created to support a robust feature flag ecosystem using cloud native technologies. OpenFeature will provide a unified API and SDK, and a developer-first, cloud-native implementation, with extensibility for open source and commercial offerings.
+## What is feature flagging?
+
+Essentially, feature flags allow application behavior to be altered without a deployment of new code. In the most basic case, a feature flag that returns a boolean value can control which code path is executed based on the configured value of such a flag in the _flag management system_. The value of the flag is evaluated at runtime, meaning that the behavior of the application can be adjusted without any code changes. This can be used to decouple the release of new features from deployment.
```

Comment:
```suggestion
Feature flags allow application behavior to be altered without a deployment of new code. In the most basic case, a feature flag can return a value (e.g. a boolean) that controls which code path is executed based on the configured value of the flag in the _flag management system_. The value of the flag is evaluated at runtime, meaning that the behavior of the application can be adjusted without any code changes. This can be used to decouple the release of new features from deployment.
```